Makefile

```diff
@@ -148,10 +202,27 @@ flash-hail:
 	LIBTOCK_PLATFORM=hail cargo run --example $(EXAMPLE) $(features) \
 		--target=thumbv7em-none-eabi $(release) -- --deploy=tockloader
 
+# ============================================
+# i.MX.RT 1050
+# ============================================
+
+.PHONY: imxrt1050
+imxrt1050:
+	LIBTOCK_PLATFORM=imxrt1050 cargo run --example $(EXAMPLE) $(features) \
+		--target=thumbv7em-none-eabi $(release) 
+	mkdir -p target/tbf/imxrt1050
+	cp target/thumbv7em-none-eabi/release/examples/$(EXAMPLE).tab \
+		target/thumbv7em-none-eabi/release/examples/$(EXAMPLE).tbf \
+		target/tbf/imxrt1050
+
+# ============================================
+# BBC micro:bit v2
+# ============================================
+
 .PHONY: microbit_v2
 microbit_v2:
 	LIBTOCK_PLATFORM=microbit_v2 cargo run --example $(EXAMPLE) $(features) \
-		--target=thumbv7em-none-eabi $(release)
+		--target=thumbv7em-none-eabi $(release) 
```

Comment:
Nit: clean up the extra space at the end of this line.

There are several other places in this PR to remove an extraneous space after `$(release)` as well.